pool_configs/vot.json

```diff
@@ -0,0 +1,87 @@
+{
+    "enabled": false,
+    "coin": "vot.json",
+
+    "address": "",
+    "_comment_address": "a transparent address to send coinbase rewards to and to transfer to zAddress.",
+
+    "zAddress": "",
+    "_comment_zAddress": "a private address used to send coins to tAddress.",
+
+    "tAddress": "",
+    "_comment_tAddress": "transparent address used to send payments, make this a different address, otherwise payments will not send",
+
+    "walletInterval": 2.5,
+
+    "rewardRecipients": {
+        "": 0.2
+    },
+
+    "paymentProcessing": {
+	"minConf": 10,
+        "enabled": false,
+        "paymentMode": "prop",
+        "_comment_paymentMode":"prop, pplnt",
+        "paymentInterval": 20,
+        "minimumPayment": 0.1,
+        "maxBlocksPerPayment": 3,
+        "daemon": {
+            "host": "127.0.0.1",
+            "port": 19332,
+            "user": "testuser",
+            "password": "testpass"
+        }
+    },
+
+    "tlsOptions": {
+        "enabled": false,
+        "serverKey":"",
```

Comment:
I also fixed lack of space after the colon on several of these in the other PR.

Reply:
sounds good. i went through and cleaned up the way it looked anyways and a few other things like the correct port numbers. let me know if vot is correct.